client/src/components/About.js

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 
 const About = () => {
-  return <div>About</div>;
+  return <div>A diversity-focused recruitment platform.</div>;
```

Comment:
I think we need to have the "About" title in the page and then a description of the project.

Reply:
Maybe we can delay that until the team + PM have decided what actually goes in the about page? (more generally, I think we should consider approving changes unless they go against the acceptance criteria of the ticket)